scripts/ci/manage-operator.sh

```diff
@@ -7,7 +7,7 @@ else
 fi
 
 WAS_ALREADY_PAIRED_FILE=/tmp/toolchain_e2e_already_paired
-OWNER_AND_BRANCH_LOCATION=${OWNER_AND_BRANCH_LOCATION:-codeready-toolchain/toolchain-cicd/master}
+OWNER_AND_BRANCH_LOCATION=${OWNER_AND_BRANCH_LOCATION:-codeready-toolchain/toolchain-e2e/master}
```

Comment:
you can drop this one too and simplify the related section to something like this:
```
start_collecting_logs() {
  scripts/ci/collect-logs.sh -n ${NAMESPACE} &
  scripts/ci/collect-logs.sh -n "openshift-operator-lifecycle-manager"  &
}
```
but TBH, the collect-logs.sh is not really used. I can give you an explanation about it during our 1-1

Reply:
Deal, thanks!

Reply:
Yes, good catch :+1: @mfrancisc

Reply:
Yes, thanks!! I saw that we need to update `run_wait_until_is_installed` as well. I am just testing locally (finally i am not getting `bad record MAC` anymore today 🙏 )